pkg/chaosdaemon/pb/chaosdaemon.proto

```diff
@@ -10,6 +10,12 @@ service ChaosDaemon {
 
   rpc FlushIpSet(IpSetRequest) returns (google.protobuf.Empty) {}
   rpc FlushIptables(IpTablesRequest) returns (google.protobuf.Empty) {}
+
+  rpc ContainerKill(ContainerRequest) returns (google.protobuf.Empty) {}
+}
+
+message ContainerRequest{
+  string container_id = 1;
 }
```

Comment:
Any design for this RPC request? For future scalability, I think `kill` is just one type of action to containers. So maybe we can add a field like `type` or `action`

Reply:
yes. I think maybe other signal can be set in type

Reply:
But this is ok currently. We can leave that change to future PR

Reply:
> Any design for this RPC request? For future scalability, I think `kill` is just one type of action to containers. So maybe we can add a field like `type` or `action`

I agree with your option, @fewdan you can add a field to represent the chaos action.